### ISSUE TITLE
Wrong quickstart example in the book

### ISSUE THREAD

**Issue by mladedav** [cucumber-rs/cucumber#200]
The quickstart uses this function to start the example (only the path is important here):
```
    futures::executor::block_on(AnimalWorld::run("/tests/features/book"));
```
Then when this is converted to tokio at the end of the quickstart, that path also has a leading slash.

This does not work for me with cucumber saying `Failed to parse: Failed to parse feature: Could not read path: /tests/features/book`, however when the leading slash is deleted everything works as expected.

I wanted to check first whether this should work (and maybe does not because of my system?) or the leading slashes should be rdeleted.

I am running rust 1.58 on Windows 10, cucumber 0.11.2.

**Comment by ilslv** [comment]
@mladedav 

> I am running rust 1.58 on Windows 10

Yeah, thats makes sense, because afaik all current maintainers are using macOS or Linux. If thats ok for you, I'll happily review your PR fixing this and maybe also adding windows target to the `test-book` CI job.

**Comment by tyranron** [comment]
This has been fixed in #202 and now is tested on CI.
